### PR TITLE
Update guidance on assets

### PR DESCRIPTION
**Pull request by lauraghiorghisor-tw**
[Trello](https://trello.com/c/YIs8YZmF/3146-fix-live-assets-on-superseded-editions)

### CODE REVIEW

File: source/manual/manage-assets.html.md.erb [hunk 1]
```diff
@@ -19,6 +19,8 @@ For example, for `https://assets.publishing.service.gov.uk/government/uploads/up
 
 ## Remove an asset
 
+Some applications, such as Whitehall, might require setting a redirect, rather than deleting the asset. Look out for replacements, or unpublished state of the parent edition, in the publishing application, and set the `replacement_id` or `redirect_url` in asset manager accordingly.
```
Comment (ChrisBAshton) [inline source/manual/manage-assets.html.md.erb:22]: Could we expand this out into its own "Redirect an asset" section instead? And maybe add some blurb at the top of the doc, where we explain when to redirect and when to simply remove.

I'd love to see this section expanded out with code examples (e.g. how to set the `redirect_url`? Whether to set `replacement_id` or `redirect_url` - which is preferred? etc)

Comment (lauraghiorghisor-tw) [reply source/manual/manage-assets.html.md.erb:22]: There already is a redirect a section but I wanted to put it in the delete section as a reminder for people like me who totally forgot there are other states and just fixed all zendesk ticket assets with deletion 🙈

I can do an intro section with an explanation of the various states of assets, at least what I understand of it so far. And move this bit in that section, at the top of the file.

The [redirect section](https://github.com/alphagov/govuk-developer-docs/blob/f635b86f7833b73fa437ab6e55bfcaf2e138c2c6/source/manual/manage-assets.html.md.erb#L48) uses a bit of an odd syntax for the rake task. [Rake task ](https://github.com/alphagov/asset-manager/blob/bf1189b102a2c6d104fe5c9df601c3fa801d895d/lib/tasks/assets.rake#L15)seems alright. Never used it though.

Comment (ChrisBAshton) [reply source/manual/manage-assets.html.md.erb:22]: Sounds good - let me know if you want a hand with syntax etc.

**Review by ryanb-gds — COMMENTED** [review]
I don't know if the developer docs are quite the right place for some of this stuff - would probably be good to move the heavier stuff into the Asset Manager or Whitehall docs and then link to them from here. I feel the developer manuals are more aimed at "I want to quickly fix a problem without having to understand the whole of asset publishing, what do I do?"

**Comment by lauraghiorghisor-tw** [comment]
On the back of comments: 
- Extracted the fields summary out into this PR - https://github.com/alphagov/asset-manager/pull/1568/files

**Review by minhngocd — APPROVED** [review]
Looks great - thank you 🙏

**Pull request merged**